bin/pycheck

```diff
@@ -20,6 +20,14 @@ cd "$(dirname "$0")/.."
 py_folders=(ci misc/python)
 
 try bin/pyactivate --dev -m mypy "${py_folders[@]}"
+if ! try_last_failed; then
+    # mzworkflows.py files need to be passed to mypy individually to avoid
+    # duplicate module warnings. We don't check these if the previous `mypy`
+    # invocation failed, though, or we'll get a ton of duplicate warnings.
+    for file in $(git_files "**/mzworkflows.py"); do
```

Comment:
Can we just typecheck all the python stuff in test ? I can do this in  a follow-up PR .